policy-controller/src/main.rs

```diff
@@ -7,5 +7,15 @@ static GLOBAL: jemallocator::Jemalloc = jemallocator::Jemalloc;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    #[cfg(feature = "rustls-tls")]
+    {
+        if rustls::crypto::aws_lc_rs::default_provider()
+            .install_default()
+            .is_err()
+        {
+            anyhow::bail!("No other crypto provider should be installed yet");
+        }
+    }
```

Comment:
Pretty sure the parent braces are unnecessary here.

```suggestion
      if rustls::crypto::aws_lc_rs::default_provider()
          .install_default()
          .is_err()
      {
          anyhow::bail!("No other crypto provider should be installed yet");
      }
```